dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -561,6 +561,10 @@ private static String outerClassName(JavaClass sourceJavaClass) {
     return null;
   }
 
+  boolean isClassSymbolReferenceUsed(String sourceClassName, ClassSymbol classSymbol) {
+    return !isUnusedClassSymbolReference(sourceClassName, classSymbol);
```

Comment:
Oh. Do we need both? I think positive method names are usually better than negative ones. Where else is `isUnusedClassSymbolReference` used, and can it be switched?

Reply:
Updated.

>  positive method names are usually better than negative ones

Yes, especially when we negate the result.